### PR TITLE
Revised antibody tests UI

### PR DESCRIPTION
**Pull request by juliatan**
# Description

Notion: https://www.notion.so/joinzoe/COVID-tests-Antibody-test-UI-amendment-614352b6bd6542d18e9ceba32b383d34
Figma: https://www.figma.com/file/7TEcX7f6rw3Z0WYR4TpX3R/ZOE-COVID-Study-(App)---MASTER?node-id=12017%3A157720

This initial PR is for the "Add new test" UI changes only. PR is starting to get quite large so decided to open for review now (but please don't merge yet!).

I will branch off this and continue working on the "Update test" feature once I complete the backend changes needed.

The one outstanding change that still needs to be done is to bold certain words in a radio button label.

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Out of scope and potential follow-up

[comment]: <> (Are there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?)

## Screenshots

Simulator Screen Shot - iPhone 11 - 2021-09-15 at 12 18 01 Simulator Screen Shot - iPhone 11 - 2021-09-15 at 12 18 06 Simulator Screen Shot - iPhone 11 - 2021-09-15 at 12 18 13

## Checklist (for submission)

- [ ] Analytics/tracking events have been added (if relevant)
- [ ] Added new screen(s) to `src/features/screens.ts` and new Redux slice(s) to `mocks/mockedInitialState.ts` (if relevant)

## Checklist (for reviewers)

- [x] Checked against Figma designs (if relevant)
- [ ] Checked that data has been successfully saved to the backend (if relevant)

## Checklist (for merger)

- [ ] Make sure the right merge strategy is chosen, commit merge into master and squash merge into develop.

### CODE REVIEW

File: src/appConfig.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export default {
   consentVersionUK: 'v2.1',
   consentVersionUS: 'v1',
 
-  covidTestVersion: '2.0.0',
+  covidTestVersion: '2.1.0',
```
Comment (aikewoody) [inline src/appConfig.ts:12]: I am wondering where we use this for exactly?

File: src/features/covid-tests/components/CovidTestThrNumberInfoModal.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      onRequestClose={props.onRequestClose}
+      testID="covid-test-thr-number-info-modal"
+      visible={props.visible}
+    >
+      <HeaderText style={styles.modalTitle}>{i18n.t('covid-test.thr-number-modal.title')}</HeaderText>
+      <Image resizeMode="contain" source={thrNumberInstructions} style={styles.thrImage} />
+      <LightText>{i18n.t(`covid-test.thr-number-modal.description`)}</LightText>
+    </Modal>
+  ) : (
+    <></>
```
Comment (aikewoody) [inline src/features/covid-tests/components/CovidTestThrNumberInfoModal.tsx:33]: Isn't it better to return null here?

File: src/features/covid-tests/fields/CovidTestInvitedQuestion.tsx [hunk 3]
```diff
@@ -25,46 +40,125 @@ interface ICovidTestInvitedQuestion<P, Data> extends React.FC<P> {
...
+      <YesNoField
+        required
+        error={formikProps.touched.invitedToTest && formikProps.errors.invitedToTest}
+        label={i18n.t('covid-test.question-invite-to-test')}
+        onValueChange={formikProps.handleChange('invitedToTest')}
+        selectedValue={formikProps.values.invitedToTest}
+        testID="covid-test-invited-question"
+      />
+      {props.test && isOldVersionAntibodyInviteTest(props.test) ? (
+        <></>
```
Comment (aikewoody) [inline src/features/covid-tests/fields/CovidTestInvitedQuestion.tsx:62]: Same as the previous file, with conditional rendering we use `null` to not render something.

**Review by aikewoody — APPROVED** [review]
Great work on this! A lot of complicated case specific code for this feature

File: src/appConfig.ts (hunk 1 above)
Comment (juliatan) [reply src/appConfig.ts:12]: It's saved to the backend so data scientists know that there's been a change in the UI. We also use it in this function `isPostDHSCRevisedVersion` to check what UI we should display to user when they are updating an old/new ZOE-invited antibody test.

**Review by aikewoody — APPROVED** [review]

**Pull request merged**